### ISSUE TITLE
8장 naver_movie_review.ipynb  에서 from sklearn.utils.fixes import loguniform를 from scipy.stats import loguniform 로 바꾸니 동작이 되네요

### ISSUE THREAD

**Issue by buskingsue** [rickiepark/python-machine-learning-book-3rd-edition#2]
구글 코랩에서 사이킷런 1.3버전을 사용하는 경우 'log'를 'log_loss'로 바꾸어 사용하고  
 from sklearn.utils.fixes import loguniform 을
 from scipy.stats import loguniform 로 바꾸니 동작이 되네요
주피터 노트북은 어떻게 되는지 모르겠습니다
 sgd = SGDClassifier(loss='log_loss', random_state=1)
![sci](https://github.com/rickiepark/python-machine-learning-book-3rd-edition/assets/138076776/158640fd-49e9-45a9-af07-082f325ba6fa)

**Comment by rickiepark** [comment]
네 수정되었습니다.
감사합니다.
